common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -135,13 +135,26 @@ object PageElement {
         m.price
       )).toList
 
+      case Comment => (for {
+        c <- element.commentTypeData
+        html <- c.html
+      } yield {
+        CommentBlockElement(
+          body = CommentCleaner.getBody(html),
+          avatarURL = CommentCleaner.getAvatar(html),
+          dateTime = CommentCleaner.getDateTime(html),
+          permalink = c._4.getOrElse(""), // originalUrl field
```

Comment:
you can just write `c.originalUrl`, the same for the ones below

Reply:
Ah my autocomplete didn't like it! Will try thanks.